src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Encryption/IdentityFilesystemWriter.php

```diff
@@ -35,12 +35,12 @@ public function __construct(RemoteVettingConfiguration $configuration)
     /**
      * Writes identity data to a data store. The data should be passed as a string.
      *
-     * @param string $data
+     * @param string[] $data
      */
-    public function write($data)
+    public function write(array $data)
```

Comment:
I don't think the serialization is the responsibility of the writer?

Reply:
Good point changed it back to let it simply write the string input